plugins/shopify/access_key.go

```diff
@@ -0,0 +1,52 @@
+package shopify
+
+import (
+	"context"
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func AccessKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.CLIToken,
+		DocsURL: sdk.URL("https://shopify.dev/docs/themes/tools/cli"),
+		//ManagementURL: sdk.URL("https://admin.shopify.com/store/{YOUR_STORE_ID}/apps/theme-kit-access"), // Can't support unless URL supports variable URLs
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Token,
+				MarkdownDescription: "Token used to authenticate to Shopify Themes.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 39,
+					Prefix: "shptka_",
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+		},
+		Importer: importer.NoOp(),
+		// Can't Implement Shopify environment parsing as the files are stored by project directory instead of a fixed location
```

Comment:
We can move this comment on top of the `Importer:` field to point out that it refers to the importing logic,